src/transformers/training_args.py

```diff
@@ -791,6 +791,11 @@ class TrainingArguments:
 
         eval_use_gather_object (`bool`, *optional*, defaults to `False`):
             Whether to run recursively gather object in a nested list/tuple/dictionary of objects from all devices. This should only be enabled if users are not just returning tensors, and this is actively discouraged by PyTorch.
+
+        use_liger (`bool`, *optional*, defaults to `False`):
```

Comment:
I would recommend changing to `use_liger_kernel` to match the product name

Reply:
+1